app/services/exports/export_product_drives_csv_service.rb

```diff
@@ -3,29 +3,35 @@ class ExportProductDrivesCSVService
     extend ItemsHelper
     HEADERS = ["Product Drive Name", "Start Date", "End Date", "Held Virtually?", "Quantity of Items", "Variety of Items", "In Kind Value"]
 
-    def self.generate_csv(product_drives)
+    def self.generate_csv(product_drives, organization)
       CSV.generate do |csv|
-        csv << HEADERS
+        csv << generate_headers(organization)
 
         product_drives.each do |drive|
           csv << generate_row(drive)
         end
       end
     end
 
-    private_class_method(
-      # The following indentation is idiotic, but it's what rubocop wants
-      def self.generate_row(drive)
-      [
-        drive.name.to_s,
-        drive.start_date.strftime("%m-%d-%Y").to_s,
-        drive.end_date&.strftime("%m-%d-%Y").to_s,
-        (drive.virtual ? "Yes" : "No").to_s,
-        drive.donation_quantity.to_s,
-        drive.distinct_items_count.to_s,
-        dollar_value(drive.in_kind_value).to_s
-      ]
+    class << self
+      private
+
```

Comment:
@dorner  Could you weigh in on if there is a reason that one of these idioms is superior to the other?   [I find private_class_method to be really clear/explicit.  ]

Reply:
just sharing the reason I used `class << self`, I think  its easier to batch the private class methods together when you have multiple of them, and it also does a better job in separating instance and class methods.

Reply:
If you're up for the change -- I suggest making this exporter look like all the other exporters where it is an actual instance instead of class methods. Then the caller (ProductDrivesController) needs to change `.generate_csv(@product_drives)` to `.new(@product_drives).generate_csv` and otherwise everything else gets a bit more normal -- you can set these up as regular private instance methods.

See ExportRequestService for example.

Reply:
sure, let me make this change

Reply:
Hey @awwaiid, I have made the requested changes, thanks for the suggestion.

Reply:
Heh I actually prefer class / module methods over instance ones, because they're much easier to work with in tests. If you don't need to instantiate a new object, it's best not to. :smile: 

Re `private_class_method`, I rarely if ever see this in the wild. It's more of a metaprogramming construct than a way to define an actual method. `class << self` is good if a lot / most of the methods are class-level. If some are class-level and some are instance-level, I actually prefer

```ruby
private

def self.generate_row(drive)
```

since it's much more obvious which methods are class or instance-level just by looking at them.

Reply:
Yep, I'm fine with that (hence my note that my comment was not blocking).